blender/LilySurfaceScrapper/Scrappers/TexturesOneScrapper.py

```diff
@@ -24,6 +24,7 @@
 class TexturesOneScrapper(AbstractScrapper):  
     source_name = "Textures.one"
     home_url = "https://www.textures.one"
+    scrapped_type = "MATERIAL & WORLD"
```

Comment:
Does this work? I bet it rather make it not available in the Material tab nor in the World tab. Anyway if we want to handle multi-type scrappers, we'll better do it using sets:

```python
scrapped_types = {"MATERIAL", "WORLD"}
```

And I fear usign the same class for both types may break things, so I would be more in favor of a separate class that inherits from this one, as discussed previously. Something like:

```python
class TexturesOneWorldScrapper(TexturesOneScrapper):
    scrapped_type = "WORLD"
    # and that's it!
```

Reply:
Yes, it does work, rn the code is using a simple is "MATERIAL" in String check.
IMO it's not necessary to have two different classes, because the Textures one scrapper doesn't do anything that is specific to Material or World. If that changes in the future we could still split them in two classes, but we'd also have to keep in mind that they share the same cache via inheritance.
Generally I think it would be beneficial if the `scrapped_type` was an argument on `canHandleUrl()`, so that classes could decide for themselves. With listing, there should probably be an attribute like `hidden = True` or so that will prevent Scrappers from being shown in the UI